src/system.c

```diff
@@ -1675,6 +1675,38 @@ void SySetGapRootPath( const Char * string )
 
 #endif
 
+/****************************************************************************
+**
+*F syLongjmp( <jump buffer>, <value>)
+** Perform a long jump
+**
+*F SignalSyLongjmp( <func> )
+** Register a function to be called before longjmp is called.
```

Comment:
Also, please document the return value, e.g. "Returns 1 upon success, and 0 if the handler could not be registered"